cli/src/commands/duplicate.rs

```diff
@@ -70,18 +70,20 @@ pub(crate) struct DuplicateArgs {
     /// commit)
     #[arg(
         long,
-        short,
+        short = 't',
```

Comment:
Nit: I'm not really sure about `-t` for `onto`, there's `-o` which will make it like a compiler invocation but I don't have a better suggestion.